Lesson9/organizer9/scheduler/forms.py

```diff
@@ -0,0 +1,60 @@
+# scheduler/forms.py
+# forms.py should contain fields input check
+from django import forms
+from django.core.exceptions import ValidationError
+from .models import Note
+
+
+class NoteForm(forms.Form):
+    title = forms.CharField(max_length=20)
+    msg = forms.CharField(max_length=100)
+    assignee = forms.CharField(max_length=100, required=False)
+    e_mail = forms.EmailField(required=False)
+
+    def clean_title(self):
+        cln_title = self.cleaned_data['title']
+
+        if len(cln_title.split(' ')) < 2:
+            raise ValidationError("Use two words Title")
```

Comment:
Three words in the title are also valid, isn't it? The message should be consistent with the validation.